requirements.txt

```diff
@@ -0,0 +1,4 @@
+markdown-it-py==3.0.0
+mdurl==0.1.2
+Pygments==2.17.2
+rich==13.7.1
```

Comment:
If you look at the relevant GitHub Action workflow, you will see that these packages are installed as part of the following step:

    pip install pelican[markdown] shot-scraper

Rather than taking the form of a `requirements.txt` file, I think it's better for this step (and other necessary steps) to be documented, which I just did via: 74fb998

Given that, I think this `requirements.txt` file can be removed.